readthedocs/builds/tasks.py

```diff
@@ -93,6 +127,15 @@ def route_for_task(self, task, args, kwargs, **__):
             )
             return project.build_queue
 
+        # Check if the project has the faster builds feature enabled
+        if project.has_feature(TYPE_FASTER_BUILDS):
```

Comment:
has_feature refers to feature flags, not to constants from subscriptions.

Reply:
I fixed this locally, but somehow lost the change 🙃